pkg/network/protocols/postgres/telemetry.go

```diff
@@ -9,10 +9,11 @@ package postgres
 
 import (
 	"fmt"
-
-	"github.com/DataDog/datadog-agent/pkg/network/protocols/postgres/ebpf"
+	pgebpf "github.com/DataDog/datadog-agent/pkg/network/protocols/postgres/ebpf"
 	libtelemetry "github.com/DataDog/datadog-agent/pkg/network/protocols/telemetry"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"reflect"
+	"strconv"
 )
```

Comment:
imports should be sorted into groups

```go
import (
    <std>

    <3rd party>

    <internal imports>
)
```

Reply:
Guy, Golang sets includes and sorts by name automatically without my intervention. Even if I change order the Goland sorts back by name after saving file.

Reply:
https://github.com/DataDog/datadog-agent/pull/28503#discussion_r1722798509

Reply:
done